packages/coinstac-ui/app/render/components/consortium/consortium-results.js

```diff
@@ -1,33 +1,43 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Alert } from 'react-bootstrap';
+import { setExpandedComputation } from '../../state/ducks/consortia-page';
 
 import ConsortiumResult from './consortium-result';
 
-export default class ConsortiumResults extends Component {
+class ConsortiumResults extends Component {
   constructor(props) {
     super(props);
 
-    // TODO: `state` holds collapsed/expanded bool for remote results. Move to redux.
-    this.state = {
-      expanded: [true],
-    };
+    this.goToResults = this.goToResults.bind(this);
   }
 
-  toggleCollapse(index) {
-    if (index in this.state.expanded) {
-      this.state.expanded[index] = !this.state.expanded[index];
-    } else {
-      this.state.expanded[index] = true;
+  componentWillMount() {
+    const { initialComputationId, setExpandedComputation } = this.props;
+
+    setExpandedComputation(initialComputationId);
+  }
+
+  componentDidMount() {
+    this.goToResults();
+  }
+
+  goToResults() {
+    if (document.querySelector('#results')) {
+      document.querySelector('#results').scrollIntoView();
     }
+  }
 
-    this.setState({
-      expanded: this.state.expanded,
-    });
+  toggleCollapse(computationId) {
+    const { setExpandedComputation, expandedComputation } = this.props;
+
+    if (computationId !== expandedComputation) setExpandedComputation(computationId);
```

Comment:
Surprised the linter’s not barking at this, although the style guide doesn’t appear to have strong opinion around [blocks](https://github.com/airbnb/javascript#blocks). Please add explicit braces around `if`s to keep things readable and consistent:

```diff
- if (computationId !== expandedComputation) setExpandedComputation(computationId);
- else setExpandedComputation('');
+ if (computationId !== expandedComputation) {
+   setExpandedComputation(computationId);
+ } else {
+   setExpandedComputation('');
+ }
```

Or, use a ternary operator:

```js
setExpandedComputation(
  computationId !== expandedComputation ? computationId : ''
);
```